pkgs/development/compilers/gcc/7/default.nix

```diff
@@ -325,6 +331,10 @@ stdenv.mkDerivation ({
     ]
     ++ optional (targetPlatform == hostPlatform && targetPlatform.libc == "musl") "--disable-libsanitizer"
     ++ optional (targetPlatform.isAarch64) "--enable-fix-cortex-a53-843419"
+    ++ optionals (targetPlatform.isPower && targetPlatform.libc == "glibc") [
+      "--with-long-double-128"
+      "--enable-softfloat"
```

Comment:
It would be nice to leverage https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/gcc/common/platform-flags.nix for this

Reply:
AFAICS platform-flags only has `--with-xyz=abc` with a select few 'xyz'. Unless you'd like me to extend it …

(Also, __float128 being required is only for the combination glibc, little endian, 64 bits … I might actually want to check endianness and word size too …)